.github/renovate.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":prHourlyLimit4",
+    ":prConcurrentLimit20",
+    ":pinAllExceptPeerDependencies",
```

Comment:
What's the reasoning behind this? It doesn't look like we currently have any `peerDependencies`.

Reply:
ah, yes, while you don't use any `peerDependencies` you do utilise the `engines` property.

see: https://docs.renovatebot.com/presets-default/#pinallexceptpeerdependencies

if you prefer a specific `packageRule` over using this preset to skip pinning the `engines` property I can change it to that.

Reply:
Oh, I wasn't aware of the correlation with `engines`. In that case I think it's fine to keep it like it is. Thanks for the clarification!